core/store/key_store_test.go

```diff
@@ -22,47 +22,45 @@ func TestCreateEthereumAccount(t *testing.T) {
 	assert.NoError(t, err)
 
 	files, _ := ioutil.ReadDir(store.Config.KeysDir())
-	assert.Len(t, files, 2)
+	assert.Len(t, files, 1)
```

Comment:
Because we now need to explicitly call MustAddRandomKeyToKeystore and before it would do that automatically in test setup I guess?

Reply:
Before it was dumping the fixture key